controllers/component_build_controller_pac.go

```diff
@@ -681,12 +703,22 @@ func (r *ComponentBuildReconciler) UnconfigureRepositoryForPaC(log logr.Logger,
 
 // generatePaCPipelineRunForComponent returns pipeline run definition to build component source with.
 // Generated pipeline run contains placeholders that are expanded by Pipeline-as-Code.
-func generatePaCPipelineRunForComponent(component *appstudiov1alpha1.Component, pipelineSpec *tektonapi.PipelineSpec, additionalPipelineParams []tektonapi.Param, onPull bool) (*tektonapi.PipelineRun, error) {
+func generatePaCPipelineRunForComponent(
+	component *appstudiov1alpha1.Component,
+	pipelineSpec *tektonapi.PipelineSpec,
+	additionalPipelineParams []tektonapi.Param,
+	onPull bool,
+	targetBranch string) (*tektonapi.PipelineRun, error) {
+
 	var targetBranches []string
 	if component.Spec.Source.GitSource != nil && component.Spec.Source.GitSource.Revision != "" {
```

Comment:
move this to selection of target branch -> it's strange that we pass targetBranch to the function but it's not being used.

Reply:
Done